frontend/components/Dashboard/PostCard/MyFeedPostCard.tsx

```diff
@@ -69,4 +71,4 @@ const PostCard: React.FC<Props> = ({ id, title, body, image, author }) => {
   )
 }
 
-export default PostCard
+export default MyFeedPostCard
```

Comment:
I wonder if we should give these slightly more generic names, just in case their purpose changes in the future, such as `PostCardStack` for the one where the info is stacked vertically in the feed currently and `PostCardInline` or something along those lines. We could use flexbox naming conventions of `PostCardCol` and `PostCardRow`. That feels like it would be better to me, since these could easily one day be repurposed and they are quite flexible, so giving them page-specific names feels a wee bit wonky, but let me know what you think 😊

Reply:
Chatted in person about the approach - new post component at `frontend/components/Dashboard/Posts/Post.tsx` and is currently used on the profile page. I'll update the My Feed page once the `<Post>` component supports avatars in a future PR 😄

Reply:
Oh I feel like that should perhaps be called `PostCard` then like before (the new one I mean). `Post` feels too generic to me and we already have a `Post` component for an entire post that someone is viewing, which feels more correct to me 🤔 What was wrong with `PostCard`?

Reply:
Yup yup! it's `PostCard` 😊